app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Models/Place.java

```diff
@@ -1,58 +1,62 @@
 package ithaca_transit.android.cornellappdev.com.ithaca_transit.Models;
 
-public final class Place extends LocationObject{
+import com.google.gson.annotations.SerializedName;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.Optional;
+
+import ithaca_transit.android.cornellappdev.com.ithaca_transit.R;
+import ithaca_transit.android.cornellappdev.com.ithaca_transit.Singleton.Repository;
+
+public final class Place{
     private String detail;
-    private String detailKey;
+
+    @SerializedName("lat")
+    private Double latitude;
+
+    @SerializedName("long")
+    private Double longitude;
     private String name;
-    private String nameKey;
     private String placeID;
```

Comment:
should be `placeId`, refer to https://source.android.com/setup/contribute/code-style#treat-acronyms-as-words

Reply:
No SIR that is wrong. we need to keep the models the same as what backend sends us